subsys/caf/modules/CMakeLists.txt

```diff
@@ -15,3 +15,6 @@ zephyr_sources_ifdef(CONFIG_CAF_CLICK_DETECTOR click_detector.c)
 zephyr_sources_ifdef(CONFIG_CAF_LEDS leds.c)
 
 zephyr_sources_ifdef(CONFIG_CAF_SENSOR_SAMPLER sensor_sampler.c)
+
+zephyr_sources_ifdef(CONFIG_CAF_BLE_SMP ble_smp.c)
```

Comment:
We could consider making CAF a Zephyr library - it could be done as separate PR (discussed offline)